From 95b917037f6af4f686b7dd08645cc4cffb95b43c Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Fri, 3 Sep 2021 00:05:13 +0100
Subject: .bashrc 000: [PATCH 120/191] Formatting

---
 .bashrc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -138,6 +138,6 @@ mvcd () {
     fi
 }
 cd_newest_sisterfolder () {
-    cd "$(find .. -maxdepth 1 -type d -print0 |grep -zZ -v '^\.*$'|xargs -0 -s 129023 -n 129023 --exit --no-run-if-empty ls -dt|head -1)$"
+    cd "$(find .. -maxdepth 1 -type d -print0 | grep -zZ -v '^\.*$' | xargs -0 -s 129023 -n 129023 --exit --no-run-if-empty ls -dt | head -1)$"
 }
 cd_newest () {
