From ae04978b1accd5c87b343314928988b26eaed489 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 26 Aug 2020 23:10:40 +0100
Subject: .bashrc 000: [PATCH 084/191] Move CHJHOSTNAME to 'right place', too

---
 .bash_profile | 5 +++++
 .bashrc       | 5 -----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -13,11 +13,6 @@ shopt -s checkwinsize
 # make less more friendly for non-text input files, see lesspipe(1)
 [ -x /usr/bin/lesspipe ] && eval "$(lesspipe)"
 
-# $HOSTNAME is apparently magical (apparently reading from the
-# kernel), thus use another env var to keep actual hostname
-# definition, for chroots:
-export CHJHOSTNAME="$(head -1 /etc/hostname)"
-
 # set a fancy prompt (non-color, unless we know we "want" color)
 case "$TERM" in
 xterm-color)
