From 9172035a88746f4e65ae8e1f1065672bbee570ae Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Nov 2017 18:12:53 +0000
Subject: .bashrc 000: [PATCH 026/191] cleanup

---
 .bashrc | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -76,7 +76,7 @@ cdnewdir() {
 };
 mvcdnewdir() {
     if [ "$#" -gt 1 ]; then
-        mvnewdir "$@" && cd "${!#}"   # ${!#}: Tip 16Nov02 1544 von #bash
+        mvnewdir "$@" && cd "${!#}"
     else
         echo At least two arguments expected
     fi
