From 01d1f9c5d0e61bdee6bbd5a3f2db1049fc72e9c7 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Thu, 11 Mar 2021 16:15:55 +0000
Subject: .bash_profile: [PATCH 098/191] .bash_profile: add cj-qemucontrol to PATH

---
 .bash_profile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.bash_profile b/.bash_profile
index 6f5ff60..0c4cc48 100644
--- a/.bash_profile
+++ b/.bash_profile
@@ -20,7 +20,7 @@ export CHJHOSTNAME="$(head -1 /etc/hostname)"
 # the default umask is set in /etc/login.defs
 # umask 002
 
-PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/opt/chj/bin:/opt/chj/cj-git-patchtool:/opt/chj/git-sign/bin:/opt/chj/chjize/bin
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/opt/chj/bin:/opt/chj/cj-git-patchtool:/opt/chj/git-sign/bin:/opt/chj/cj-qemucontrol/bin:/opt/chj/chjize/bin
 
 # set PATH so it includes user's private bin if it exists
 if [ -d ~/bin ] ; then
