From 5d1ebc04448350b46eec0486fc017caac13cbcf5 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 1 Sep 2024 13:08:47 +0200
Subject: .bashrc 007: [PATCH 180/191] .bashrc: cdn, cdnn, cdnnn: pass -a option to lastdir

Removing the -a by default.
---
 .bashrc | 43 ++++++++++++++++++++++++++++++++++---------
 1 file changed, 34 insertions(+), 9 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -165,4 +178,10 @@ mvcd () {
     fi
 }
 cdnn () {
+    local opts
+    opts=""
+    if [ "${1-}" = "-a" ]; then
+        opts="-a"
+        shift
+    fi
     if [ $# -eq 0 ]; then
