From ebd6f9a9b7f076972f4ff6c5c2cef7a95a511c1e Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Tue, 15 Jun 2021 04:11:44 +0100
Subject: .bashrc 001: [PATCH 103/191] .bashrc: show exit code on failing commands

---
 .bashrc | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -20,7 +29,7 @@ shopt -s checkwinsize
 # set a fancy prompt (non-color, unless we know we "want" color)
 case "$TERM" in
 xterm-color|xterm)
-    PS1='\[\033[01;32m\]\u@$CHJHOSTNAME\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+    PS1='$(__ps1_show_exitcode)\u@$CHJHOSTNAME\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
     ;;
 *)
     PS1='\u@$CHJHOSTNAME:\w\$ '
