From ae04978b1accd5c87b343314928988b26eaed489 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 26 Aug 2020 23:10:40 +0100
Subject: .bash_profile 000: [PATCH 084/191] Move CHJHOSTNAME to 'right place', too

---
 .bash_profile | 5 +++++
 .bashrc       | 5 -----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/.bash_profile b/.bash_profile
--- a/.bash_profile
+++ b/.bash_profile
@@ -9,6 +9,11 @@ if [ "`readlink -f "$PWD"`" = "`readlink -f "$HOME"`" ]; then
     PWD=$HOME
 fi
 
+# $HOSTNAME is apparently magical (apparently reading from the
+# kernel), thus use another env var to keep actual hostname
+# definition, for chroots:
+export CHJHOSTNAME="$(head -1 /etc/hostname)"
+
 # the default umask is set in /etc/login.defs
 # umask 002
 
