From c0dc80eec89295c7be39bb600ddee1df86126131 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Nov 2017 18:14:31 +0000
Subject: .bash_profile: [PATCH 027/191] more move to .bash_profile, beautify

---
 .bash_profile | 9 +++++++++
 .bashrc       | 2 --
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/.bash_profile b/.bash_profile
index 058ab03..f4bffe6 100644
--- a/.bash_profile
+++ b/.bash_profile
@@ -24,6 +24,8 @@ if [ -d ~/local/bin ] ; then
     PATH=~/local/bin:"${PATH}"
 fi
 
+
+# --- General env setup -------
 unset LESSOPEN
 unset LESSCLOSE
 # not running Gnome anymore, for some reason this env var is set, why
@@ -33,13 +35,20 @@ export COLUMNS
 export HISTSIZE=1500
 
 
+# --- Machine specific env setup: -------
+export ALSARECDEV='sysdefault:CARD=C1100'
+
 ulimit -S -v 1200000
 
+
+# --- Personal env setup: -------
 export EDITOR=e
 export BROWSER="chromium-chrissbx -- --new-window"
 export EMAIL='ch@christianjaeger.ch'
 export LANG=en_GB.UTF-8
 
+
+# --- End -------------------------------------
 # include .bashrc if it exists
 if [ -f ~/.bashrc ]; then
     . ~/.bashrc
