From eef5b2df89adfd1a6f323b3f56d0b6741802d065 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 7 Jun 2020 22:02:13 +0100
Subject: [PATCH 076/191] .bashrc: have u etc. take optional argument

---
 .bashrc | 19 ++++++++++++++-----
 1 file changed, 14 insertions(+), 5 deletions(-)

diff --git a/.bashrc b/.bashrc
index 391e56f..98096a0 100644
--- a/.bashrc
+++ b/.bashrc
@@ -59,11 +59,20 @@ fi
 #    . /etc/bash_completion
 #fi
 
-u () { cd ..; }
-uu () { cd ../..; }
-uuu () { cd ../../..; }
-uuuu () { cd ../../../..; }
-uuuuu () { cd ../../../../..; }
+maybe_cd () {
+    if [ $# = 1 ]; then
+        cd "$1"
+    elif [ $# -gt 1 ]; then
+        echo "too many arguments"
+	false
+    fi
+}
+
+u () { cd ..; maybe_cd "$@"; }
+uu () { cd ../..; maybe_cd "$@"; }
+uuu () { cd ../../..; maybe_cd "$@"; }
+uuuu () { cd ../../../..; maybe_cd "$@"; }
+uuuuu () { cd ../../../../..; maybe_cd "$@"; }
 les () { less "$@"; }
 c () { cd "$@"; }
 cdnewdir () {
-- 
2.47.3

