From 5d1e3b88b0dd50afe387b90459bd9fef5e2b1c84 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Thu, 10 Dec 2020 04:08:34 +0000
Subject: .bashrc 000: [PATCH 087/191] .bashrc: clean up with regards to ~/.bashrc_local

---
 .bashrc | 22 +++++++++++++++-------
 1 file changed, 15 insertions(+), 7 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -2,6 +2,10 @@
 # see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
 # for examples
 
+# NOTE: you can use ~/.bashrc_local for local changes (i.e. those that
+# shouldn't make it to the Git repo), it is included at the end of
+# this file.
+
 
 # If not running interactively, don't do anything
 [ -z "$PS1" ] && return
