From 76908cc15329c38f89a2af25f6c1a08bacc807d7 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 16 Oct 2019 22:22:01 +0000
Subject: [PATCH 071/191] Turn off new Git editor 'waiting' warning

---
 .chj-home/init | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.chj-home/init b/.chj-home/init
index 60aaa47..0b8249d 100755
--- a/.chj-home/init
+++ b/.chj-home/init
@@ -59,6 +59,9 @@ cat <<EOF > .gitconfig
 [core]
 	excludesfile = ~/.gitignore_global
 
+[advice]
+        waitingForEditor = false
+
 #[gpg]
 #	program = gpg+scrypt
 
-- 
2.47.3

