From 2ae56d5d60c850ac9bb76e6c91d5d7e8a2a2e482 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Tue, 18 Nov 2025 10:49:51 +0100
Subject: [PATCH 189/191] .bashrc: add cdnnnnnn and cdnnnnnnn

---
 .bashrc | 28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/.bashrc b/.bashrc
index e7f6249..fcac8c1 100644
--- a/.bashrc
+++ b/.bashrc
@@ -243,6 +243,34 @@ cdnnnnn () {
         cdnewdir "$@"
     fi
 }
+cdnnnnnn () {
+    local opts
+    opts=""
+    if [ "${1-}" = "-a" ]; then
+        opts="-a"
+        shift
+    fi
+    if [ $# -eq 0 ]; then
+	cd_newest 5 "$opts"
+    else
+	cd_newest 4 "$opts"
+        cdnewdir "$@"
+    fi
+}
+cdnnnnnnn () {
+    local opts
+    opts=""
+    if [ "${1-}" = "-a" ]; then
+        opts="-a"
+        shift
+    fi
+    if [ $# -eq 0 ]; then
+	cd_newest 6 "$opts"
+    else
+	cd_newest 5 "$opts"
+        cdnewdir "$@"
+    fi
+}
 
 _cgd_ () {
     local gd_="$1"
-- 
2.47.3

