From 757137ff0e73a7a6a308b4e629800203eea0476e Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Tue, 26 Sep 2017 12:18:48 +0100
Subject: .Xresources 000: [PATCH 010/191] remove more ancient entries

---
 .Xresources | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/.Xresources b/.Xresources
--- a/.Xresources
+++ b/.Xresources
@@ -22,5 +22,4 @@ emacs23*toolbarlines: 0
 
 emacs*geometry: 80x65
 
-! emacs23*font: -Adobe-Courier-Medium-R-Normal--17-120-100-100-M-100-ISO8859-1
 emacs23.font: -Adobe-Courier-Medium-R-Normal--17-120-100-100-M-100-ISO8859-1
