From 5d1ebc04448350b46eec0486fc017caac13cbcf5 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 1 Sep 2024 13:08:47 +0200
Subject: .bashrc 003: [PATCH 180/191] .bashrc: cdn, cdnn, cdnnn: pass -a option to lastdir

Removing the -a by default.
---
 .bashrc | 43 ++++++++++++++++++++++++++++++++++---------
 1 file changed, 34 insertions(+), 9 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -151,7 +156,9 @@ mvcd () {
 cd_newest () {
     local n
     n="${1-0}"
-    cd "$(_ls_newest "$n" .)"
+    local opts
+    opts="${2-}"
+    cd "$(_ls_newest "$n" "$opts" .)"
 }
 cdat() {
     cdnewdir "$(dat --day --week "$@")"
