From 6ef5c25d64273c41d5ee5cf3775f2a3ecad7d9ad Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Mon, 27 Nov 2017 17:38:22 +0000
Subject: home-init 001: [PATCH 041/191] home-init: only run once

---
 home-init | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/home-init b/home-init
--- a/home-init
+++ b/home-init
@@ -81,6 +87,7 @@ lesskey
 chmod a+wxt,g+s DROP
 touch .ssh/authorized_keys
 chmod go-w .ssh/authorized_keys
+touch .home-init-done
 
 set +x
 
