From d6a7f736b21577f4a3164f161884b0af5ab54810 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 24 Jul 2022 13:14:18 +0100
Subject: .bashrc: [PATCH 151/191] .bashrc: cgd*: accept index argument

---
 .bashrc | 61 +++++++++++++++++++++++++++++++++++++++++++++++----------
 1 file changed, 51 insertions(+), 10 deletions(-)

diff --git a/.bashrc b/.bashrc
index 838dd21..86ef3ee 100644
--- a/.bashrc
+++ b/.bashrc
@@ -173,33 +173,74 @@ cdnnn () {
         cdnewdir "$@"
     fi
 }
+
 _cgd_ () {
-    local res=$1
-    # I forgot how to do this with builtins:
+    local gd_="$1"
+    shift
+    if [ $# = 0 ]; then
+        echo "Please give name-regex (and optionally index into result list)" >&2
+        return 1
+    fi
+    local last="${@:$#:1}"
+    local index=""
+    local args
+    declare -a args
+    if printf '%s' "$last" | egrep -q '^[0-9]+$'; then
+        index="$last"
+        args=("${@:1:$(( $# - 1 ))}")
+    else
+        args=("$@")
+    fi
+    local res
+    if ! res="$("$gd_" "${args[@]}")"; then
+        return 1
+    fi
+    # (XX do these with builtins?)
     if [ -n "$res" ]; then
-        if [ "$(printf '%s' "$res" | wc -l)" -eq 0 ]; then
+        local len
+        len0="$(printf '%s' "$res" | wc -l)"
+        if [ "$len0" -eq 0 ]; then
             cd "$res"
         else
-            printf '*** Need exactly one result, found:\n%s\n' "$res"
-            false
+            if [ -n "$index" ]; then
+                if [ "$index" -le "$len0" ]; then
+                    local item
+                    if ! item="$(printf "%s" "$res" | skiplines "$index")"; then
+                        return 1
+                    fi
+                    if ! item="$(printf "%s" "$item" | head -1)"; then
+                        return 1
+                    fi
+                    cd "$item"
+                else
+                    echo "Index is pointing behind last item" >&2
+                    false
+                fi
+            else
+                local numbered
+                numbered=$(printf '%s' "$res" | linenumbers)
+                printf '*** More than one result, please give index:\n%s\n' "$numbered" >&2
+                false
+            fi
         fi
     else
-        echo "Nothing found."
+        echo "Nothing found." >&2
         false
     fi
 }
 cgd () {
-    _cgd_ "$(gd "$@")"
+    _cgd_ gd "$@"
 }
 cgdi () {
-    _cgd_ "$(gdi "$@")"
+    _cgd_ gdi "$@"
 }
 cgi () {
-    _cgd_ "$(gi "$@")"
+    _cgd_ gi "$@"
 }
 cgii () {
-    _cgd_ "$(gii "$@")"
+    _cgd_ gii "$@"
 }
+
 cdt () {
     if checkcreate-tmp-owner-dir; then
 	cd "/tmp/$USER"
