From b108a22573430b063d80ffadfb11fd85ebb560dd Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 9 Jul 2017 03:04:13 +0100
Subject: .bashrc 000: [PATCH 007/191] obsolete

---
 .bashrc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -83,7 +83,6 @@ alias les=less
 alias le=zless
 #^ zless?  egal ja ebe ?. nun. -- nope nid egal. cat file.gz|less doesn't work, zless does.
 alias c=cd
-alias cdsp="cd ~/Projekte/spielzeug"
 cdnewdir() {
     if [ "$#" -eq 1 ]; then
         mkdir "$1" && cd "$1"
