From c7d942e546df485c4d8d7b9bddd9484d951ca17e Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Nov 2017 18:03:12 +0000
Subject: .bash_profile 000: [PATCH 020/191] clean up PATH

---
 .bash_profile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.bash_profile b/.bash_profile
--- a/.bash_profile
+++ b/.bash_profile
@@ -17,7 +17,7 @@ if [ -f ~/.bashrc ]; then
     . ~/.bashrc
 fi
 
-PATH=/root/TEMP/cj-git-patchtool:/home/chris/NEUE/bin/:/home/chris/Projekte/spielzeug/bin:/opt/chj/bin:/usr/local/sbin:/usr/local/bin:/usr/local/Gambit-C/current/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
+PATH=/opt/chj/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
 
 # set PATH so it includes user's private bin if it exists
 if [ -d ~/bin ] ; then
