From 7909e48f85a167ea1c61ccc95cdb9d51e818b24e Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Nov 2017 22:06:22 +0000
Subject: .bash_profile 000: [PATCH 028/191] source .bash_profile_local

---
 .bash_profile | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/.bash_profile b/.bash_profile
--- a/.bash_profile
+++ b/.bash_profile
@@ -49,7 +49,9 @@ export LANG=en_GB.UTF-8
 
 
 # --- End -------------------------------------
-# include .bashrc if it exists
+if [ -f ~/.bash_profile_local ]; then
+    source ~/.bash_profile_local
+fi
 if [ -f ~/.bashrc ]; then
-    . ~/.bashrc
+    source ~/.bashrc
 fi
