From 114eb72c07538a5a86044ec6306827a7b476fad8 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sat, 25 Nov 2017 18:11:00 +0000
Subject: .bash_profile 000: [PATCH 025/191] more move to .bash_profile, incl. HISTSIZE

---
 .bash_profile | 2 ++
 .bashrc       | 4 +---
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/.bash_profile b/.bash_profile
--- a/.bash_profile
+++ b/.bash_profile
@@ -29,6 +29,8 @@ unset LESSCLOSE
 # not running Gnome anymore, for some reason this env var is set, why
 # no idea, XX.
 unset GNOME_KEYRING_CONTROL
+export COLUMNS
+export HISTSIZE=1500
 
 
 ulimit -S -v 1200000
