From 087495679c102270e73e1fcdafefc0df595bd366 Mon Sep 17 00:00:00 2001
From: "Christian Jaeger (Rustaceans)" <ch@christianjaeger.ch>
Date: Thu, 30 Nov 2017 19:30:42 +0000
Subject: .bash_profile 000: [PATCH 050/191] .bash_profile: stop the X beep

---
 .bash_profile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.bash_profile b/.bash_profile
--- a/.bash_profile
+++ b/.bash_profile
@@ -39,6 +39,10 @@ export HISTSIZE=1500
 
 ulimit -S -v 3200000  # note: can override in .bash_profile_local
 
+if [ -n "${DISPLAY-}" ]; then
+    xset -b
+fi
+
 
 # --- Personal env setup: -------
 # see ~/.bash_profile_local
