From 9c495d33b41919d596f355a5bf98c6d256f186f0 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Tue, 15 Jun 2021 04:27:32 +0100
Subject: .bashrc: [PATCH 105/191] .bashrc: show exit code on a separate line

---
 .bashrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.bashrc b/.bashrc
index ff2706c..81c3a21 100644
--- a/.bashrc
+++ b/.bashrc
@@ -20,7 +20,7 @@ shopt -s checkwinsize
 __ps1_show_exitcode () {
     local v=$?
     if [ "$v" -ne 0 ]; then
-	echo -ne '\033[01;41m'"$v"'\033[00m \033[01;32m';
+	echo -ne '\033[01;41m'"$v"'\033[00m\n\033[01;32m';
     else
 	echo -ne '\033[01;32m';
     fi
