From 497bc6425624f556c278479259129b55e0c30db3 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 7 Jun 2020 22:07:41 +0100
Subject: .bashrc 001: [PATCH 077/191] .bashrc: make u etc. "transaction safe" (kind of)

---
 .bashrc | 25 +++++++++++++++++++------
 1 file changed, 19 insertions(+), 6 deletions(-)

diff --git a/.bashrc b/.bashrc
--- a/.bashrc
+++ b/.bashrc
@@ -68,11 +68,24 @@ maybe_cd () {
     fi
 }
 
-u () { cd ..; maybe_cd "$@"; }
-uu () { cd ../..; maybe_cd "$@"; }
-uuu () { cd ../../..; maybe_cd "$@"; }
-uuuu () { cd ../../../..; maybe_cd "$@"; }
-uuuuu () { cd ../../../../..; maybe_cd "$@"; }
+_cd_then () {
+    local to="$1"; shift
+    if [ $# = 1 ]; then
+        local old=$(pwd)
+        cd "$to" && cd "$1" || cd "$old"
+    elif [ $# -gt 1 ]; then
+        echo "too many arguments"
+	false
+    else
+	cd "$to"
+    fi
+}
+
+u () { _cd_then .. "$@"; }
+uu () { _cd_then ../.. "$@"; }
+uuu () { _cd_then ../../.. "$@"; }
+uuuu () { _cd_then ../../../.. "$@"; }
+uuuuu () { _cd_then ../../../../.. "$@"; }
 les () { less "$@"; }
 c () { cd "$@"; }
 cdnewdir () {
