From 3e1fb7a5d1ec1cc6ea3e5c405132496bc707a4d7 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Sun, 26 Nov 2017 00:21:32 +0000
Subject: .gitconfig: [PATCH 038/191] .gitconfig: cleanup

---
 .gitconfig | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/.gitconfig b/.gitconfig
index 3596608..67e3e19 100644
--- a/.gitconfig
+++ b/.gitconfig
@@ -1,22 +1,20 @@
-
-[diff "cj"]
-	command = _cj-git-tkdiff
-
 [user]
 	email = ch@christianjaeger.ch
 	name = Christian Jaeger
 
+[diff "cj"]
+	command = _cj-git-tkdiff
+
 [merge]
 	conflictstyle = diff3
 	#renameLimit = 2000
+
 [core]
 	excludesfile = ~/.gitignore_global
-[credential]
-	username = pflanze
 
 #[gpg]
 #	program = gpg+scrypt
 
 #[push]
 #	default = simple
-#wow old git can*not* deal with this
+# old git can*not* deal with this
